qcodes/instrument_drivers/rohde_schwarz/ZNB.py

```diff
@@ -270,9 +398,44 @@ def __init__(self, parent: 'ZNB', name: str, channel: int,
                            get_cmd=f'SENS{n}:SWE:TIME?',
                            get_parser=float,
                            unit='s')
+        self.add_parameter(name='sweep_type',
+                           get_cmd=f'SENS{n}:SWE:TYPE?',
+                           set_cmd=self._set_sweep_type,
+                           val_mapping={'Linear': 'LIN\n',
+                                        'Logarithmic': 'LOG\n',
+                                        'Power': 'POW\n',
+                                        'CW_Time': 'CW\n',
+                                        'CW_Point': 'POIN\n',
+                                        'Segmented': 'SEGM\n',
+                                        },
+                           docstring="Allows switching the default linear VNA "
+                                     "sweep type to other types. Note that "
+                                     "at the moment only the linear and "
+                                     "CW_Point modes have supporting "
+                                     "measurement parameters."
+                           )
+        self.add_parameter(name='cw_frequency',
+                           get_cmd=f'SENS{n}:FREQ:CW?',
+                           set_cmd=self._set_cw_frequency,
+                           get_parser=float,
+                           vals=vals.Numbers(self._parent._min_freq + 0.5,
+                                             self._parent._max_freq - 10),
+                           docstring="Similar to center, except that this is "
+                                     "used when VNA sweep type is set to "
+                                     "CW_Point mode."
```

Comment:
Please make this docstring stand on its own rather than haing to read another docstring to understand what this does